test/typescript/terraform-cloud/test.ts

```diff
@@ -9,6 +14,66 @@ if (!TERRAFORM_CLOUD_TOKEN) {
   console.log("TERRAFORM_CLOUD_TOKEN is undefined, skipping authed tests");
 }
 
+process.on("uncaughtException", (err) => {
+  console.dir(err);
+  console.log("UNHANDLED");
+  process.exit(1);
+});
+
+process.on("unhandledRejection", (err) => {
+  console.dir(err);
+  console.log("UNHANDLED");
+  throw err;
+});
+
+function startHttpProxy(mock: jest.Mock): Promise<{
```

Comment:
nit, maybe we can put this into the with proxy describe block so it's closer to it's usage

Reply:
Hmm, yeah that's a good point. For me it was adding noise while reading the test cases, but I guess just having to have this function in general makes it harder, not the place

Reply:
Should I put it in a separate file? 🤔 I'd be down for that.